comp/networkpath/npscheduler/npschedulerimpl/npscheduler.go

```diff
@@ -66,13 +66,13 @@ func newNoopNpSchedulerImpl() *npSchedulerImpl {
 	return &npSchedulerImpl{enabled: false}
 }
 
-func newNpSchedulerImpl(epForwarder eventplatform.Forwarder, logger log.Component, sysprobeYamlConfig config.Reader) *npSchedulerImpl {
-	workers := sysprobeYamlConfig.GetInt("network_path.workers")
-	pathtestInputChanSize := sysprobeYamlConfig.GetInt("network_path.input_chan_size")
-	pathtestProcessChanSize := sysprobeYamlConfig.GetInt("network_path.process_chan_size")
-	pathtestTTL := sysprobeYamlConfig.GetDuration("network_path.pathtest_ttl")
-	pathtestInterval := sysprobeYamlConfig.GetDuration("network_path.pathtest_interval")
-	flushInterval := sysprobeYamlConfig.GetDuration("network_path.flush_interval")
+func newNpSchedulerImpl(epForwarder eventplatform.Forwarder, logger log.Component, agentConfig config.Reader) *npSchedulerImpl {
+	workers := agentConfig.GetInt("network_path.workers")
+	pathtestInputChanSize := agentConfig.GetInt("network_path.input_chan_size")
```

Comment:
Isn't this `network_path.collector.input_chan_size`?

Reply:
@hmahmood Yes, I need to update the configs to use `network_path.collector.workers/input_chan_size/etc` and `network_path.connections_monitoring.enabled`. Will to it today/tomorrow.

Is `network_path.connections_monitoring` a good name?